modules/cnf-du-updating-crio.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-updating-crio_{context}"]
+
+= Updating CRI-O
```

Comment:
As titled, I expected to read about replacing the software on the RHEL or RHCOS host with an update to the CRI-O daemon.

Maybe "CRI-O configuration for workload partitioning"